internal/cliwrapper/cliwrapper.go

```diff
@@ -15,12 +15,19 @@ import (
 type cliWrapper struct {
 	podmanFullPath string
 	logger         log.Logger
+	runtimeContext []string
 }
 
-func NewCliWrapper(fullPath string, logger log.Logger) CliWrapper {
+func NewCliWrapper(fullPath string, logger log.Logger, context string) CliWrapper {
+	// Prepand the podman argument
```

Comment:
"Prepend" ... although that's not an accurate description of either the next line(s) or of the function. This constructs a new CLI wrapper object, and you've added the capability to specify a non-default connection parameter. (Which will, in time, become a podman option.)

Reply:
Ah this was a copy/paste relic from a previous failed implementation, updated.